v2_1/ws/rest/docs/4_3_structural_queries.md

```diff
@@ -80,7 +80,7 @@ Furthermore, a keyword may be used:
 
 Keyword | Scope | Description
 --- | --- | ---
-all | itemID | Returns all items belonging to the item scheme
+`*` | itemID | Returns all items belonging to the item scheme
 
 The following rules apply:
 
```

Comment:
Line 159: should be  http://ws-entry-point/structure/codelist/BIS,ECB/CL_FREQ

Reply:
Done. Thanks for spotting this, @dosse :)